src/inmanta/app.py

```diff
@@ -620,6 +620,74 @@ def export(options: argparse.Namespace) -> None:
     summary_reporter.print_summary_and_exit(show_stack_traces=options.errors)
 
 
+def validate_logging_config_parser_config(
+    parser: argparse.ArgumentParser, parent_parsers: abc.Sequence[ArgumentParser]
+) -> None:
+    """
+    Config parser for the validate-logging-config command.
+    """
+    parser.add_argument(
+        "-e",
+        dest="environment",
+        help="The environment id to be used as context variable in logging config templates. If not specified,"
+        " 0c111d30-feaf-4f5b-b2d6-83d589480a4a will be used.",
+        default="0c111d30-feaf-4f5b-b2d6-83d589480a4a",
```

Comment:
This feels very arbitrary. Can't we use something like "<env-id>", or does it need to be a valid uuid?

Reply:
This is based on a suggestion that Wouter made (see comment above). I think it's a good to have a uuid by default, because like this the name of the logger will be as it is in reality. It's indeed a very arbitrary default value, but it has two main advantages:

* The user doesn't need to specify a uuid him-/herself.
* The uuid is stable.